Portfolio/index.html

```diff
@@ -1,177 +1,205 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <section id="Home"></section>
+    <h1>Doe.</h1>
+    <div class="navbar">
+        <ul>
+            <li>ABOUT</li>
+            <li>SKILLS</li>
+            <li>PROJECTS</li>
+            <li>REVIEWS</li>
+
+        </ul>
+        <img src="/phone.icon.png" alt="Css Img" width="20px" height="20px">
     </div>
 
-        <header>
-        <h1>About Me</h1>
-    </header>
-    <hr>
+    <div class="btn">
+        <button>
+            <bold>Contact Me</bold>
+        </button>
+    </div>
 
 
-    <section>
-        <h2>Introduction</h2>
-        <div class="img">
-            <img src="/1701186972765.jpg" alt="Omparth" width="10%" height="10%">
+    <section class="abtme" id="abtme">
+        <div>
+            <h1>John Doe</h1>
+            <img src="/johnDoe.png" alt="Css Img" width="300px" height="300px">
+            <h1><b>Front End Developer</b></h1>
+            <h3>Get To Know me</h3>
+            <p><b>I'm a seasoned front-end web developer with a passion for crafting exceptional digital experiences.
+                    With over 2 years of hands-on experience, i bring a deep proficiency in HTML, CSS, JavaScript,
+                    TypeScript, and a keen mastery of modern frameworks like React.js, React Native, Svelte.js. I
+                    specialize in building responsive, interactive, and visually appealing user interfaces.
+                    Additionally, my proficiency in Firebase ensures secure and efficient backend integration, making me
+                    a versatile and dynamic developer ready to tackle any project</b>
         </div>
-        <p>Hello there! I'm Omparth Maruti Chougale, a passionate Software Engineer with a keen interest in Web Development. I love to code and to play games.</p>
-    </section>
+        </p>
 
-    <section>
-        <h2>My Skills</h2>
-        <ul>
-            <li>HTML</li>
-            <li>CSS</li>
-            <li>JavaScript</li>
-            <li>Python</li>
-            <li>Photoshop</li>
-        </ul>
+        <h4>
+            <div>Name:John Doe</div>
+            <div>Age:24</div>
+            <div>Experience: 2years</div>
+        </h4>
+        </div>
     </section>
 
-    <section class="Exper">
-        <h2>Experience</h2>
-        <p>I have over fresher of experience. Throughout my career, I've worked on projects related to web development.</p>
-    </section>
+    <!-- skills -->

```

Comment:
Consider adding a space between the closing 'div' tag and the opening 'img' tag on the next line for better readability.